normalize-git-url.js

```diff
@@ -18,7 +18,16 @@ module.exports = function normalize (u) {
 
   var returnedUrl
   if (parsed.pathname.match(/\/?:/)) {
-    returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '').replace(/#[^#]*$/, '')
+    if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
+      returnedUrl = u.replace(/^(.*:[^:]+):(.*)/, '$1/$2')
+      console.warn('deprecated: ' + u + ' is using an' +
```

Comment:
agree 1000%. Do you think I should just remove the warning altogether and only warn in release notes?